src/input.F

```diff
@@ -161,16 +161,16 @@ MODULE input
               nutk,nvtk,nwtk
 
 !$acc declare create(nqs1,nqs2,nqr,nqi,nqs,nqg,nnci,nncc, &
-!$acc                ptype,psolver,ngxy,ngz,nx,ny,nz,axisymm, &
+!$acc                ptype,ngxy,ngz,nx,ny,nz, &
 !$acc                nparcels,npvals,npvars,ni,nj,nk,nip1,njp1, &
-!$acc                nkm1,nkp1,ib,ie,jb,je,kb,ke,terrain_flag, &
```

Comment:
@sjsprecious It appears that axisymm and terrain_flag are also still needed on the GPU.

Reply:
@sjsprecious I have returned several additional variables to the GPU.  I think all variables that I erroneously removed from the GPU are now back.